Error Compiling for Nano

\Arduino\libraries\DDS-master\src\DDS.cpp:48:62: error: default argument given for parameter 1 of 'void DDS::startPhaseAccumulator(bool)' [-fpermissive]
 void DDS::startPhaseAccumulator(bool use_only_timer_2 = false){
                                                             ^
In file included from \Arduino\libraries\DDS-master\src\DDS.cpp:2:0:
\Arduino\libraries\DDS-master\src\DDS.h:146:8: error: after previous specification in 'void DDS::startPhaseAccumulator(bool)' [-fpermissive]
   void startPhaseAccumulator(bool use_only_timer_2 = false);
        ^
exit status 1
Error compiling for board Arduino Nano.

--- a/src/DDS.cpp
+++ b/src/DDS.cpp
@@ -45,7 +45,7 @@ void DDS::stop() {
   TCCR2B = 0;
 }
 
-void DDS::startPhaseAccumulator(bool use_only_timer_2 = false){
+void DDS::startPhaseAccumulator(bool use_only_timer_2){
     timer2only = use_only_timer_2;
 
   if(timer2only){

